includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -950,7 +946,7 @@ public function get_payment_method_ids_enabled_at_checkout( $order_id = null, $f
 		$active_payment_methods     = $this->get_upe_enabled_payment_method_statuses();
 		foreach ( $capturable_payment_methods as $payment_method_id ) {
 			$payment_method_capability_key = $this->payment_method_capability_key_map[ $payment_method_id ] ?? 'undefined_capability_key';
-			if ( false !== WC_Payments::get_payment_method_by_id( $payment_method_id ) ) {
```

Comment:
`WC_Payments::get_payment_method_by_id()` is a static method and it uses a private static variable inside. I couldn't think of a way to mock this out. So I created a wrapper function `wc_payments_get_payment_method_by_id()`  to decouple this.